src/Microsoft.Health.Checkpoints/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Resources;
+
+[assembly: CLSCompliant(false)]
```

Comment:
Out of curiosity what prevents this from being CLSCompliant(true)?

Reply:
I am not sure if there is anything specific that makes it not compliant other than us not verifying that it is compliant. The rest of the projects in this solution did this, so just wanted to be consistent.

Reply:
You can make this compliant if you wish, and you can read about the requirements [here](https://docs.microsoft.com/en-us/dotnet/standard/language-independence).

Ultimately, you'll probably just be limited by the types you can use (like no unsigned types). When I added the assembly attributes, I think each of the assemblies at the time had some conformance problem (at least as I remember it).